products/microos/main.pm

```diff
@@ -91,6 +91,15 @@ sub load_installation_tests {
     }
 }
 
+sub load_security_tests_selinux {
+    # SELinux tests for Micro OS operating system
+    # ALWAYS run following tests in sequence because of the dependencies
+    loadtest 'security/selinux/sestatus';
+    loadtest 'security/selinux/selinux_smoke';
```

Comment:
Same as above comment, selinux_smoke.pm makes some dangerous assumptions that undermine the goal here of testing what MicroOS actually has configured by default

https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/security/selinux/selinux_smoke.pm#L36 assumes permissive, for MicroOS it should be targetted

https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/security/selinux/selinux_smoke.pm#L39-L44 and https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/security/selinux/selinux_smoke.pm#L59-L61 does zypper calls - these wont work on a transactional system :)

I would recommend significantly changing this test, wrapping all that zypper stuff in a `if !is_microos` block and then adding actual useful smoke tests for file creation, access, ports, etc that work on MicroOS and regular openSUSE. Such generic additional smoke tests should be beneficial to all users of that smoke test so I wouldn't advocate for creating a MicroOS specific test module..just the current module isn't really fit for even its current stated purpose :)